asf_public_discourse_home_decarbonisation/getters/mse_getters.py

```diff
@@ -0,0 +1,113 @@
+"""
+Getter functions for MSE data.
+"""
+import pandas as pd
+from asf_public_discourse_home_decarbonisation.getters.getter_utils import load_s3_data
+from asf_public_discourse_home_decarbonisation import PROJECT_DIR
+import sys
+import logging
+
+logger = logging.getLogger(__name__)
+
+S3_BUCKET = "asf-public-discourse-home-decarbonisation"
```

Comment:
the idea is to use the base.yaml, but it wasn't working, so I defined the S3_BUCKET here temporarily.